ersilia/hub/content/catalog.py

```diff
@@ -122,7 +239,7 @@ def airtable(self):
             webbrowser.open("https://airtable.com/shrUcrUnd7jB9ChZV")
 
     def _get_all_github_public_repos(self):
-        url = "https://api.github.com/users/{0}/repos".format(GITHUB_ORG)
+        url = "https://api.github.com/users/y{0}/repos".format(GITHUB_ORG)
```

Comment:
There's an extraneous `y` here. This is causing tests to fail because it's forming an incorrect URL.

Reply:
@DhanshreeA apologies for this error, his has also been resolved in the latest commit